catalog/data-model-extensions/@sine/unlocode/0.1.0/README.md

```diff
@@ -0,0 +1,26 @@
+```
```

Comment:
Wrapping this in a code block means that the line will not be wrapped automatically. Is that really what we want here?

Reply:
That's how it's presented in the original readme, I'm ok with any option tbh